flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/functions/bridging/BridgingSqlFunction.java

```diff
@@ -110,9 +114,11 @@ public static BridgingSqlFunction of(
 			dataTypeFactory,
 			typeFactory,
 			kind,
-			identifier,
+			identifier != null ? createName(identifier) : createInlineFunctionName(definition),
```

Comment:
can we integrate the case distinction in the methods `createName()`, `createSqlIdentifier()`, `createSqlFunctionCategory()` directly? First of all, I'm not big fan of inline if/else and more importantly, we need the same logic for `BridgingSqlAggFunction` which is why the `BridgingUtils` class exists.

Reply:
sure